flair/datasets/sequence_labeling.py

```diff
@@ -562,6 +562,68 @@ def __init__(
             document_separator_token="-DOCSTART-",
             **corpusargs,
         )
+        
+        
+class STACKOVERFLOW_NER(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            tag_to_bioes: str = "NE",
+            in_memory: bool = True,
+            document_as_sequence: bool = False,
+            **corpusargs,
+    ):
+        """
+        Initialize the STACKOVERFLOW_NER corpus. The first time you call this constructor it will automatically
+        download the dataset.
+        :param base_path: Default is None, meaning that corpus gets auto-downloaded and loaded. You can override this
+        to point to a different folder but typically this should not be necessary.
+        :param tag_to_bioes: NER by default, need not be changed, but you could also select 'pos' to predict
+        POS tags instead
+        :param in_memory: If True, keeps dataset in memory giving speedups in training.
+        :param document_as_sequence: If True, all sentences of a document are read into a single Sentence object
+        """
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
+
+        """The Datasets are represented in the Conll format.
+           In this format each line of the Dataset is in the following format:
+           <word>+"\t"+<NE>"\t"+<word>+"\t"<markdown>
+           The end of sentence is marked with an empty line.
+           In each line NE represented the human annotated named entity 
+           and <markdown> represented the code tags provided by the users who wrote the posts.
+           """
+        # column format
+        columns = {0: "word", 1: "NE", 2: "word", 3: "markdown"}
```

Comment:
- Why is "word" listed twice here? If the two columns are the same, one entry of "word" is enough. 
- Can you use "ner" instead of "NE"? That's how we call it in other classes.